src/environment.cpp

```diff
@@ -256,6 +260,13 @@ void Environment::stepTimeOfDay(float dtime)
 	}
 }
 
+u32 Environment::getDayCount()
+{
+	MutexAutoLock lock(this->m_time_lock);
```

Comment:
:+1: in this case (where the value is only incremented and read) atomic works very well.